package.json

```diff
@@ -50,6 +50,7 @@
     "react-beautiful-dnd": "^13.0.0",
     "react-cookie": "^4.0.3",
     "react-dom": "^17.0.0",
+    "react-fast-compare": "^3.2.0",
```

Comment:
I didn't know about this package but it is good that you introduced it so we can use it in the future where we need.